conwaylib/BoardConsoleExtensions.cs

```diff
@@ -3,15 +3,17 @@
 using System.Text;
 using ConwayLib;
 using System.Drawing;
+using System.Linq;
```

Comment:
Presumably this is not needed as there are no other changes in this source file?